src/fizzbuzz.js

```diff
@@ -0,0 +1,25 @@
+// Return an array of numbers from 1 to 100.
+//
+// For multiples of three, use the string Fizz instead of the number and for multiples of five replace with Buzz.
+//
+// For numbers which are multiples of both three and five replace with FizzBuzz.
+export default function fizzBuzz(){
+
+  let fizzBuzzArray = []
+
+  for(let i = 1; i <= 100; i++){
+    if(i % 15 === 0){
```

Comment:
nice, I've always done this with `i % 3 === 0 && i%5 === 0`, but of course least common multiple. Yay high school math! :D